todos-lambda/Sources/App/lambda.swift

```diff
@@ -44,11 +44,16 @@ struct AppLambda: APIGatewayLambdaFunction {
         router.get("/") { _, _ in
             return "Hello"
         }
+
+        // When enabling Lambda function URLs, CORS can be configured at the AWS level, and there's
```

Comment:
You should make it clear this is APIGateway (maybe add a link), otherwise this is cool.

Reply:
@adam-fowler so I also thought this is APIGateway when I first saw it, but it turns out it's not exactly API Gateway (at least not publicly):

https://docs.aws.amazon.com/lambda/latest/dg/urls-configuration.html
https://www.serverlessguru.com/blog/aws-lambda-function-urls-vs-amazon-api-gateway

I could link to this section in the first link, what do you think?

https://docs.aws.amazon.com/lambda/latest/dg/urls-configuration.html#urls-cors

Reply:
We use APIGateway to access Lambda’s not function URLs. So I would have thought it would be related to this https://docs.aws.amazon.com/apigateway/latest/developerguide/how-to-cors.html. Did you check the APIGateway CORS setup after deployment?

Reply:
Out of interest what is the input to the lambda if you use a function URLs?

Reply:
> Did you check the APIGateway CORS setup after deployment?

I checked API Gateway, and there's no entry there for this function - it's all handled by Lambda:

![Safari-2024-10-27-14 35 16@2x](https://github.com/user-attachments/assets/2a11e383-5e5a-4d0a-a9c7-c07d6875ae45)

Here's where it's defined:

![Safari-2024-10-27-14 36 46@2x](https://github.com/user-attachments/assets/a0dc12cf-8a5a-46a3-ad09-24784ac3d3eb)

> Out of interest what is the input to the lambda if you use a function URLs?

If I understand the question correctly, the input is still a `APIGatewayV2Request`.